SWLOR.Game.Server/Feature/SpawnDefinition/DantooineResourceSpawnDefinition.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System.Collections.Generic;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Service.SpawnService;
+
+namespace SWLOR.Game.Server.Feature.SpawnDefinition
+{
+    public class DantooineResourceSpawnDefinition : ISpawnListDefinition
+    {
+        private readonly SpawnTableBuilder _builder = new();
+
+        public Dictionary<string, SpawnTable> BuildSpawnTables()
+        {
+            DanLake();
```

Comment:
Please fully spell out Dantooine on all of these spawn tables for consistency and clarity's sake.